appengine-standard-java8/helloworld-gae-launch-dataflow/README.md

```diff
@@ -58,6 +58,20 @@ detailed instructions.
     sed -i "s/YOUR_BUCKET_NAME/$BUCKET/" src/main/java/com/example/appengine/java8/LaunchDataflowTemplate.java
     ```
 
+1. In the `pom.xml`, update the [App Engine Maven Plugin](https://cloud.google.com/appengine/docs/standard/java/tools/maven-reference) 
+with your Google Cloud Project Id:
+
+```
+<plugin>
+  <groupId>com.google.cloud.tools</groupId>
+  <artifactId>appengine-maven-plugin</artifactId>
+  <version>2.2.0</version>
+  <configuration>
+    <projectId>myProjectId</projectId>
+    <version>GCLOUD_CONFIG</version>
```

Comment:
You may wish to explain that "GCLOUD_CONFIG" is a special version that asks Google to auto-generate a version for you.